packages/cli-kit/src/private/node/ui/components/Table/Row.tsx

```diff
@@ -0,0 +1,55 @@
+import ScalarDict from './ScalarDict.js'
+import {Column} from './Column.js'
+import {Box, Text} from 'ink'
+import React from 'react'
+
+interface RowProps<T extends ScalarDict> {
+  line: string
```

Comment:
I was a bit confused by the naming of this prop at first. What do you think about renaming it to something like `fillerCharacter`? Or `fillerChar`? Because in case you pass `" "` it's not really a line.